### ISSUE TITLE
The Tracking Protection functionality breaks if it's disabled on more than 3 websites in a row

### ISSUE THREAD

**Issue by Softvision-RemusDranca** [MozillaReality/FirefoxReality#3103]
**[Affected Versions]:**
- Firefox Reality version 10-140931839 (a360c20).

**[Affected Platforms]:**
- HTC Vive Plus - Android 7.1.2
- Pico Neo 2 - Android - 8.1.0
- Oculus Quest - Android 7.1.1

**[Prerequisites]:**
- The headset is turned on and the app is opened.
- Tracking protection is enabled.

**[Steps to reproduce]:**                                        
1. Navigate to a website.
2. Click the shield icon from the Awesomebar
3. Click the "Disable" button from the pop-up. 
4. Redo the steps above for other 2 websites.
5. Observe the behavior.

**[Expected results]:**
- The Tracking Protection is successfully disabled for all websites above and added to the Exception list.

**[Actual results]:**
- The Tracking Protection is not disabled for the last added website.
OR
- In some cases it's disabled but existing ones in the Exceptions list are replaced.

**[Additional Notes]:**
- Related to #3062.
- Attached is a screen recording of the issue: [link](https://drive.google.com/open?id=1P1uXSY0ae04mV-gRyDJ111g1tAgra3L-).

**Comment by bluemarvin** [comment]
This seems to work in Fenix Nightly.

**Comment by bluemarvin** [comment]
This is caused by https://github.com/MozillaReality/FirefoxReality/blob/master/app/src/common/shared/org/mozilla/vrbrowser/browser/content/TrackingProtectionStore.java#L103
What happens is:

1) The  `mContentBlockingController` is cleared
2) A list is made from `position` to `position + count`
3) The `mContentBlockingController` block list is restored with the list

This drops all exceptions not in the `position` to `position + count` range.

**Comment by Softvision-RemusDranca** [comment]
I have verified this issue and it's no longer reproducible with the latest Firefox Reality version 10-140982354 (bce60f9) build. Now, the more then 3 websites can be disabled in a row and they are added to the Exceptions list.

Tested on Oculus Go (Android 7.1.2), Oculus Quest (Android 7.1.1), HTC Vive Focus Plus (Android 7.1.2), HTC Vive Focus (Android 7.1.2) and Pico Neo 2 (Android 8.1.0).
